src/Host/Client/Impl/rtvs/R/util.R

```diff
@@ -253,6 +253,13 @@ save_to_project_folder <- function(blob_id, project_name, dest_dir) {
     invisible(call_embedded('save_to_project_folder', blob_id, project_name, path.expand(dest_dir), temp_dir));
 }
 
+save_to_temp_folder <- function (blob_id, file_name) {
+    temp_file <- paste0(tempdir(), '/', file_name);
```

Comment:
It uses R User directory. If TMPDIR, TMP, TEMP, are checked first, if none of them are defined it uses R_USER